### PR TITLE
Fixes missing/duplicate form fields/property names

### PR DESCRIPTION
**Pull request by knod**

### CODE REVIEW

**Comment by knod** [comment]
Branch it came from: https://github.com/knod/cliff-effects-1/tree/prop-typos

File: src/forms/previousExpenses.js [hunk 1]
```diff
@@ -308,7 +308,8 @@ const ExpensesFormContent = function ( props ) {
             <li>Monthly payment on accumulated medical bills (regular monthly payments on a bill that was previously incurred). The allowance may include only the amount expected to be paid in the coming 12 months.</li>
           </ul>
           <IntervalColumnHeadings type={type}/>
-          <CashFlowRow {...merge( {generic: 'DisabledAssistance'}, sharedProps )}> Disabled/Handicapped/Elderly assistance </CashFlowRow>
+          <CashFlowRow {...merge( {generic: 'DisabledOrElderlyMedicalCosts'}, sharedProps )}> Disabled/Elderly medical expenses </CashFlowRow>
+          <CashFlowRow {...merge( {generic: 'MembersMedicalCosts'}, sharedProps )}> Other members' medical expenses </CashFlowRow>
```
Comment (shp54) [inline src/forms/previousExpenses.js]: Rather than using `merge`, rewrite both as: `<CashFlowRow {...sharedProps} generic='MembersMedicalCosts'>`

Comment (knod) [reply src/forms/previousExpenses.js]: Considering the changes that have happened in the meantime, I think we may just get rid of this and redo these changes after #77 has been merged already.

**Comment by knod** [comment]
Dropping this issue till PR #77 has been resolved.

**Pull request closed without merging**